src/commons/api/roas.rs

```diff
@@ -280,22 +286,186 @@ impl Ord for RoaDefinition {
     }
 }
 
-impl PartialOrd for RoaDefinition {
+impl PartialOrd for RoaPayload {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
         Some(self.cmp(other))
     }
 }
 
-impl AsRef<TypedPrefix> for RoaDefinition {
+impl AsRef<TypedPrefix> for RoaPayload {
     fn as_ref(&self) -> &TypedPrefix {
         &self.prefix
     }
 }
 
+//------------ RoaConfiguration --------------------------------------------
+
+/// This type defines a saved RoaConfiguration. It includes the actual ROA
+/// payload that needs be authorized in a ROA object as well as other
+/// information that is only visible to users - i.e. an optional comment
```

Comment:
Users of what? I assume that users of a relying party won't see a ROA comment as a ROA comment is not someting in the RFCs, right? Maybe it's worth saying this is a custom Krill extension to what is covered by RFCs? (assuming I'm right of course!)

Reply:
fixed (I think)